subscriptions/subscriptions-impl/src/main/java/com/duckduckgo/subscriptions/impl/SubscriptionsManager.kt

```diff
@@ -68,17 +78,22 @@ interface SubscriptionsManager {
     /**
      * Returns the access token
      */
-    suspend fun getAccessToken(): AccessToken
+    fun getAccessToken(): AccessToken
```

Comment:
Why not suspend anymore. 
I think it should be `suspend` and probably set the dispatcher.io internally to (?)

Reply:
This is only used by the public API impl which is a suspend already and then left to the user to decide the dispatcher. Can easily change though.

Reply:
Changed it to suspend and added an withContext(io)